dts/arm/nordic/nrf51822.dtsi

```diff
@@ -154,6 +154,8 @@
 			reg = <0x4000b000 0x1000>;
 			interrupts = <11 1>;
 			status = "ok";
+			prescaler = <0>;
+			ppi-wrap = <0>;
```

Comment:
Could we have just "ppi-wrap;" property?

Reply:
I don't think we have something like that:
https://github.com/zephyrproject-rtos/zephyr/blob/9552796361a3f1d4f7a4607b1bceb2139774fc14/dts/bindings/device_node.yaml.template#L29

Reply:
How much effort would be required to add this?

Reply:
> I don't think we have something like that:

We have. It's `boolean` type. See for instance the binding for [PWM](https://github.com/zephyrproject-rtos/zephyr/blob/master/dts/bindings/pwm/nordic%2Cnrf-pwm.yaml#L30).

Reply:
fixed. Added another commit which mentions boolean type in `device_node.yaml`.